pom.xml

```diff
@@ -352,7 +352,7 @@
                 <artifactId>maven-javadoc-plugin</artifactId>
                 <version>3.4.1</version>
                 <configuration>
-                    <source>8</source>
+                    <source>17</source>
```

Comment:
nice

Reply:
Should update the javadoc source in every submodul in the project